app/helpers/sms_helper.rb

```diff
@@ -19,8 +19,9 @@ def format_sms_messages_field(sms, field)
       recips = sms.recipient_hashes(max: MAX_RECIPS_TO_SHOW).map { |r| user_with_phone(r[:user], r[:phone]) }.join('<br/>')
       extra_recipients = sms.recipient_count - MAX_RECIPS_TO_SHOW
       recips << (extra_recipients > 0 ? t('sms.extra_recipients', count: extra_recipients) : '')
+      recips.html_safe
     when "from" then
-      user_with_phone sms.sender, sms.from
+      user_with_phone(sms.sender, sms.from).html_safe
```

Comment:
Are we sure this is not an XSS vector? What if someone puts evil stuff in their user name?